app/static/src/scss/includes/_overrides.scss

```diff
@@ -1,3 +1,8 @@
 legend {
   padding-inline: 0;
 }
+
+// use of !important is not ideal but fits this purpose
```

Comment:
Having read your description and then information on !important, I see why you say its not ideal; it could override other fonts unintentionally if there was a specific different font needed somewhere else on the site.

Given the reason for needing to use this, do you think it it worth us creating a tech debt ticket for us to switch to using the npm package rather than the prebuilt minified css? I imagine there could be other benefits?

Reply:
@anthonyhashemi yes definitely, I don't think we use any other fonts at the moment but its still a side effect to be aware of and that we don't want. Having a ticket to have the npm package instead would be good.